storage/store.go

```diff
@@ -0,0 +1,38 @@
+package storage
+
+import (
+	"context"
+	"sync"
+)
+
+type Store struct {
+	mu sync.RWMutex
+	// highest block height
+	blockHeight uint64
+}
+
+// NewStore returns a new in-memory Store implementation.
+func NewStore() *Store {
+	return &Store{
+		mu: sync.RWMutex{},
+	}
+}
+
+func (s *Store) LatestBlockHeight(ctx context.Context) (uint64, error) {
+	return s.blockHeight, nil
```

Comment:
this should take a read lock

Reply:
Nice catch :100: 
Added in https://github.com/onflow/flow-evm-gateway/pull/11/commits/de90c6290e8d9827eeee0b2866749955f9787243